### DISCUSSION TITLE
Pricing is vague

### DISCUSSION THREAD

**Discussion by Kentoseth** [tolgee/tolgee-platform#1335] [Q&A]
Hello,

The pricing page says:

> 3 000 translations

(*as sum of keys multiplied by languages for every project*)

What is a 'key' and what is the actual translation limit?

It would be clearer to just specify how many characters is permitted or sentences.

**Comment by JanCizmar** [comment]
Hello, thanks for asking.

A key is a single string, which can be translated to multiple languages. In UI, a key is the row, containing values translated to multiple languages.

![image](https://user-images.githubusercontent.com/18496315/186924845-ea5ac086-b00f-4394-a538-60f35e6b019b.png)

We were considering other metrics, but this one seems like optimal for us. For us, number of translations describes the size of the project the best.

We are counting the translations this way, because we don't want the translators to reach the limit while translating. Which would happen if we counted characters or sentences. This way, space is always reserved for all keys to be translated to all languages.
